Stripe/STPPaymentContext.m

```diff
@@ -443,6 +443,69 @@ - (void)requestPayment {
             // Concrete source object, check flow to see if synchronous charge or webhook based
             // If a card source, we need to check if the user wants us to 3DS or not
 
+            STPSource *source = (STPSource *)self.selectedPaymentMethod;
```

Comment:
can we add a TODO: to update the block above where we present `STPAddCardVC` (L409)? Or just update it to present `STPAddSourceVC` depending on the config